services/worker/src/worker/dtos.py

```diff
@@ -145,7 +145,6 @@ class SplitDuckdbIndex(SplitHubFile):
     partial: Optional[bool]
     num_rows: Optional[int]
     num_bytes: Optional[int]
-    duckdb_version: str
```

Comment:
hmmm good question: do we want to remove this field? I think we can keep it for now

Reply:
We can keep it even if it is no longer used, for the moment...

Reply:
yes. The thing is all the entries in the db have it, so... let's not change that, I think, and it can be useful for stats purposes

Reply:
Done.